### ISSUE TITLE
LDAP _must_ bind

### ISSUE THREAD

**Issue by vincentkersten** [salesagility/SuiteCRM#2799]
It seems, if you use LDAP authentication the file 'SuiteCRM/modules/Users/authentication/LDAPAuthenticate/LDAPAuthenticateUser.php' only works if you allow it to bind. 

However It is quite common to _not_ bind to the openldap server. 

My suggested solution is to not bind to the ldap server if the 'bind attribute' is empty, as filled out in Admin>Password Management >Enable LDAP Authentication . 

And fill out the default ('dc') in that field which you seem to have choosen as a hidden deafult:

"Dec 15 13:35:55 brainserver slapd[2328]: conn=104578 op=0 do_bind: invalid dn (vincent)"

Hope this helps openLDAP users forward with suitecrm!

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [LDAP _must_ bind](https://trello.com/c/R44cqrIl/47-ldap-must-bind)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
